# tello_ai_ros config

# tello onboard camera frame shape
tello_frame:
  w: 720
  h: 960
  d: 3
  
gazebo_frame:
  w: 240
  h: 320
  d: 3

# my computer webcam shape
# if you want to test the face_detector_node via computer webcam
camera_frame:
  w: 480
  h: 640
  d: 3
  
  

# pixel values for goal state
face_goal:
  x: 0
  y: 0
  z: 100

# regions of interest (face_landmarks nose, left_eye, right_eye)
face_roi:
  nose: 4
  eye_l: 145
  eye_r: 374

# pixel values for goal state
pose_goal:
  x: 0
  y: -40
  l: 100

pose_gain:
  x: 0.05
  y: 0.05
  l: 0.02
  
# regions of interest (face_landmarks nose, left_eye, right_eye)
pose_roi:
  nose: 4
  eye_l: 145
  eye_r: 374
